magento/inventory
# Show warning for backordered items in cart

### Description
If "Allow quantity below 0 and notify customer" option is enabled for a source item or globally, show the corresponding warning if an item is backordered

### Fixed Issues (if relevant)
https://github.com/magento-engcom/msi/issues/717

### Manual testing scenarios
1. Simple product created with Qty=10
2. In Advanced inventory for product enable function "Backoredrs" = Allow quantity below 0 and notify customer
3. On storefront add to shopping cart Simple product Qty= 20
4. The warning about backorder should appear
